src/windows.rs

```diff
@@ -11,7 +11,7 @@ use std::ffi::c_void;
 pub use windows_sys::Win32::{
     Foundation::*,
     Graphics::{Dwm::*, Gdi::*},
-    System::{LibraryLoader::*, SystemInformation::*},
+    System::LibraryLoader::*,
```

Comment:
I removed this unused import as well (which is breaking CI on current dev branch)

Reply:
thanks